README.md

```diff
@@ -94,41 +95,34 @@ Your `build.gradle` file(s) should have the following things. Add them to what's
 already in your files as appropriate.
 
 ```groovy
-// Add the gradle plugins that are needed for Error Prone plugin support
-buildscript {
-  repositories {
-    maven {
-      url "https://plugins.gradle.org/m2/"
-    }
-  }
-  dependencies {
-    classpath "net.ltgt.gradle:gradle-errorprone-plugin:0.0.13"
-    classpath "net.ltgt.gradle:gradle-apt-plugin:0.12"
-  }
+import net.ltgt.gradle.errorprone.CheckSeverity
+
+plugins {
+  id("java")
```

Comment:
does the kts syntax work with regular groovy style buildscripts always?

Reply:
In this snippet, only the `tasks.withType` and `tasks.named` below would have to be different between Groovy and Kotlin. Everything else here is OK in both Groovy and Kotlin.

In Kotlin, it would read:
```kotlin
tasks.withType(JavaCompile::class).configureEach {
```
or (probably more idiomatic):
```kotlin
tasks.withType<JavaCompile>().configureEach {
```

---
and (not a syntax discrepancy here, but static typing vs dynamic one):
```kotlin
tasks.named<JavaCompile>("compileTestJava") {
```
for the most idiomatic version, the longer (-est) variant would be:
```kotlin
tasks.withType(JavaCompile::class.java).named("compileTestJava").configure {
```